src/database/repositories/EventRepository.js

```diff
@@ -129,6 +134,23 @@ class EventRepository extends Repository {
             start: { $regex: "^" + day }
         });
     }
+    async getArchivedEvents(archived = true, fields = null) {
```

Comment:
You dont have to have this method here, you can use getEvents(archived = false, fields = null) and use acrhived = true...

Reply:
:)))))